src/main/java/com/openshift3/client/model/build/BuildTriggerType.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package com.openshift3.client.model.build;
+
+public enum BuildTriggerType {
+	generic,
```

Comment:
are we sure that there will never be any further build trigger types? What would happen if a never API version that has a new trigger type is being parsed? will the unmarshalling code face IllegalArgumentException in Enum#valueOf()?